README.md

```diff
@@ -17,6 +17,66 @@ This tool is able to override the following attributes:
 
 For further reference on environment variables in the OpenTelemetry SDK, please read the [official specification](https://opentelemetry.io/docs/reference/specification/sdk-environment-variables/)
 
+## OpenTelemetry Attributes
```

Comment:
@cyrille-leclerc @v1v @kuisathaverat could please double check the attributes I'm adding (standardising) in this PR? 🙏